### ISSUE TITLE
Missing press effect for system restart

### ISSUE THREAD

**Issue by jpnurmi** [canonical/ubuntu-desktop-installer#2274]
We should probably delay the system restart a tiny bit to give Flutter a chance to render the press effect.

Originally reported by @hellsworth:

> When you click Restart at the end, the green Restart button doesn't have the visual effect of a button being pushed even though the restart has been initiated in the background.

**Comment by hellsworth** [comment]
Yes, otherwise the user may click the green button several times, unsure if it did anything :)
